Exiled.API/Features/Player.cs

```diff
@@ -1912,6 +1912,21 @@ public void Heal(float amount, bool overrideMaxHealth = false)
                 Health += amount;
         }
 
+        /// <summary>
+        /// Used an item for giving effect of it.
```

Comment:
Maybe "Uses an item for giving its effect."? I think the method name should be also Heal

Reply:
no because it's can use other things who are not heal like Scp244